app/src/redux/config/schema-types.ts

```diff
@@ -177,4 +179,11 @@ export interface ConfigV13 extends Omit<ConfigV12, 'version'> {
   }
 }
 
-export type Config = ConfigV13
+export interface ConfigV14 extends Omit<ConfigV13, 'version'> {
+  version: 14
+  protocols: ConfigV13['protocols'] & {
+    pinnedProtocolIds: PinnedProtocolIds | null
```

Comment:
maybe? 
recommend you to get feedback from @shlokamin @brenthagen 
```suggestion
    pinnedProtocolIds: string[] | null
```

Reply:
I defined a type in `schema-types.js` to make it easier to change to a more complex object in the future if we need to. Could totally get away with the `string[]` now though if that's better.

Reply:
`string[]` makes the most sense to me. Also, there should be no need to default (or allow) this value to be `null` right? That way this migration should initialize the config value to `[]` and we wouldn't need to handle an unnecessary null case.

Reply:
I've removed the `null` possibilities everywhere, except in selectors.ts:

```
export const getPinnedProtocolIds: (
  state: State
) => string[] | undefined = createSelector(
  getConfig,
  config => config?.protocols.pinnedProtocolIds
)
```

Since `config` itself can be undefined (I don't know why this is but it goes too deep for me to worry about) the returned `pinnedProtocolIds` could also possibly be undefined.